jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/LocalRegistry.java

```diff
@@ -52,6 +52,16 @@ public LocalRegistry(int port, String username, String password) {
   /** Starts the local registry. */
   @Override
   protected void before() throws IOException, InterruptedException {
+    startLocalRegistry();
```

Comment:
Oh, sorry. I thought this was a test class. Since this is `@Rule`, `startLocalRegistry()` sounds redundant.